RecoPixelVertexing/PixelTrackFitting/interface/PixelFitterByHelixProjections.h

```diff
@@ -10,29 +10,21 @@
 
 #include <vector>
 
-
+class TrackerTopology;
 
 class PixelFitterByHelixProjections final : public PixelFitterBase {
 public:
-  explicit PixelFitterByHelixProjections(const edm::EventSetup *es, const MagneticField *field);
+  explicit PixelFitterByHelixProjections(const edm::EventSetup *es, const MagneticField *field,
+                                         bool scaleErrorsForPhaseI, float scaleFactor);
   virtual ~PixelFitterByHelixProjections() {}
   virtual std::unique_ptr<reco::Track> run(const std::vector<const TrackingRecHit *>& hits,
                                            const TrackingRegion& region) const override;
 
 private:
-  /* these are just static and local moved to local namespace in cc .... 
-   *
-  int charge(const std::vector<GlobalPoint> & points) const;
-  float cotTheta(const GlobalPoint& pinner, const GlobalPoint& pouter) const;
-  float phi(float xC, float yC, int charge) const;
-  float pt(float curvature) const;
-  float zip(float d0, float phi_p, float curv, 
-    const GlobalPoint& pinner, const GlobalPoint& pouter) const;
-  double errZip2(float apt, float eta) const;
-  double errTip2(float apt, float eta) const;
-  */
-private:
   const edm::EventSetup *theES;
   const MagneticField *theField;
+  const bool thescaleErrorsForPhaseI;
```

Comment:
I can certainly rename the parameter and variable to `theScaleErrorsForPhase1BPix1` or similar (I'd keep "phase1" in the name as the tuning was done with phase1 pixel, and is not necessarily applicable to phase2 (and clarly not for phase0)).